app/services/ants_api/appointment.rb

```diff
@@ -48,25 +50,17 @@ def delete
       end
     end
 
+    def syncable?
+      self.class.status(application_id: @application_id)["status"] != CONSUMED
```

Comment:
peut-être qu'il faut même filtrer pour avoir uniquement les status validated ici ? genre : 
```suggestion
      self.class.status(application_id: @application_id)["status"] == 'validated'
```

Reply:
C'est vrai que l'ANTS a signalé que les autres statuts sont à considérer comme incorrects, donc de toute façon si le statut n'est pas validated, on est en train de fournir des infos sur une pré-demande qui n'existe même pas (ou qui est expirée et donc non pertinente).

Reply:
Fait ici : https://github.com/betagouv/rdv-service-public/pull/4225/commits/11ecec358e509dad49c6d75a364afa81e8ef23b0